plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -478,24 +483,17 @@ private static void handleStandardArgument(Props jobProps,
     if (jobSparkVer != null) {
```

Comment:
Can we add a comment to this method describing how we are handling various scenarios when determining the location of sparkHome and sparkConf?